.circleci/config.yml

```
@@ -30,27 +30,18 @@ commands:
           curl --silent -L --output ~/.docker/cli-plugins/docker-compose https://github.com/docker/compose-desktop/releases/download/v2.21.0-desktop.1/docker-compose-linux-x86_64
           chmod a+x ~/.docker/cli-plugins/docker-buildx
           chmod a+x ~/.docker/cli-plugins/docker-compose
-          echo "$DOCKER_PASS" | docker login --username $DOCKER_USER --password-stdin
-          docker buildx create --use --driver cloud "mozilla/default"
+          if [ -n "$DOCKER_USER" ]; then
```

Any harm in testing for both user and pass?

```suggestion
          if [ -n "$DOCKER_USER" -a -n "$DOCKER_PASS" ]; then
```

---

nope no harm 👍